concepts/05 UI Components/DateBox/02 Accessibility/20 Screen Reader Support.md

```diff
@@ -0,0 +1,3 @@
+The DateBox component supports screen readers and complies to <a href="https://www.w3.org/WAI/standards-guidelines/aria/" target="_blank">WAI-ARIA</a> standards. 
+
+To improve accessibility, use the [inputAttr](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#inputAttr) property to specify <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label" target="_blank">aria-label</a> for the component. You do not need to specify `aria-label` if you use [label](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#label).
```

Comment:
...`if you use [label](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#label).`

Is label an option? If so, can we change it to:

...`if you use the [label](/Documentation/ApiReference/UI_Components/dxDateBox/Configuration/#label) option.`